theano/gpuarray/opt.py

```diff
@@ -757,44 +779,44 @@ def local_gpua_subtensor(node, context_name):
             if len(x.clients) == 1:
                 if any([n == 'output' or any([isinstance(v.type, GpuArrayType)
                                               for v in n.inputs + n.outputs])
-                        for n, _ in node.outputs[0].clients]):
+                        for n, _ in clients[0]]):
                     return
                 else:
                     return [host_from_gpu(gpu_x.owner.op(node.outputs[0]))]
```

Comment:
Here just remove the else clause, the code on line 787 will handle that case correctly. With this change, you don't need to call op.make_node() anymore.